lib/buildkit_syntax.ml

```diff
@@ -12,5 +12,6 @@ let add arch =
     | `X86_64 | `I386 -> `X86_64
     | `Aarch64 | `Aarch32 -> `Aarch64
     | `Ppc64le -> `Ppc64le
-    | `S390x -> `S390x) in
+    | `S390x -> `S390x
+    | _ -> failwith "No support for this architecture in docker/dockerfile:experimental.") in
```

Comment:
It’s almost always better to be explicit about match cases to avoid some bugs later on
```suggestion
    | `Riscv64 -> failwith "No support for this architecture in docker/dockerfile:experimental.") in
```

Reply:
I hesitated, thanks.